src/ert/libres_facade.py

```diff
@@ -159,6 +159,9 @@ def get_number_of_iterations(self) -> int:
     def have_observations(self) -> bool:
         return self._enkf_main.have_observations()
 
+    def is_valid_runpath(self) -> bool:
+        return self._enkf_main.getModelConfig().is_valid_runpath()
```

Comment:
I think instead of making yet another method on `libres_facade` based on a stateful value in model_config, I would like `load_results_tool::is_valid_run_path` to be 
```python
def is_valid_run_path(self) -> bool:
  actives = self.facade.get_active_realizations()
  iter_0_paths + set(self.facade.get_run_paths(actives, 0))
  iter_1_paths + set(self.facade.get_run_paths(actives, 1))
  return len(iter_0_paths) == len(actives) and iter_0_paths != iter_1_paths
```

Reply:
I do agree, but would like to discuss the details regarding this. :+1: